tests/ctst/steps/replication.ts

```diff
@@ -0,0 +1,148 @@
+import { When, Then } from '@cucumber/cucumber';
+import Zenko from '../world/Zenko';
+import { createAndRunS3UtilsPod, getMongoDBConfig } from 'steps/utils/kubernetes';
+import assert from 'assert';
+import { GetObjectCommand } from '@aws-sdk/client-s3';
+import { v4 as uuidv4 } from 'uuid';
+import { getObject, headObject, getReplicationLocationConfig } from 'steps/utils/utils';
+import { safeJsonParse } from 'common/utils';
+
+When('I run the job to replicate existing objects with status {string} from bucket {string} to location {string}',
+    { timeout: 600000 },
+    async function (
+        this: Zenko,
+        sourceObjectStatus: string,
+        sourceBucket: string,
+        replicationLocation: string,
+    ) {
+        const { replicaSetHosts, username, password, database } = await getMongoDBConfig(this);
+        const { locationType } = getReplicationLocationConfig(this, replicationLocation);
+        const podManifest = {
+            apiVersion: 'v1',
+            kind: 'Pod',
+            metadata: {
+                name: `s3utils-crr-existing-${uuidv4()}`,
+                namespace: 'default',
+                labels: {
+                    app: 's3utils',
+                    script: 'crrExistingObjects.js'
+                }
+            },
+            spec: {
+                restartPolicy: 'Never',
+                containers: [
+                    {
+                        name: 's3utils',
+                        image: `ghcr.io/scality/s3utils:${this.parameters.S3UtilsTag}`,
```

Comment:
(minor comment) I believe we have the image locally and do not need to pull it from the ghcr repository? That shouldn't change anything as we never recreate the same tag though...